etc/openqa/openqa.ini

```diff
@@ -115,12 +115,16 @@
 #do_push = no
 ## whether to clone CASEDIR or NEEDLES_DIR on the web UI if that var points to a Git repo
 #git_auto_clone = yes
-## enable automatic updates of all test code and needles managed via Git (still experimental, currently still breaks scheduling parallel clusters)
-#git_auto_update = no
+## enable automatic updates of all test code and needles managed via Git
+#git_auto_update = yes
```

Comment:
this change does not look like it should be in this PR

Reply:
You can find the original commit here: https://github.com/os-autoinst/openQA/commit/e466e05e548f58bcf2303c2d9c29e6904063eb36#diff-b323be01fab1562900d8df608b80822e7ba6ca5d54dc745ca778315adbabe065R119

Now is committed with https://github.com/os-autoinst/openQA/pull/5175/commits/5ee9dd151e20ce0531d27dfd924d21e243055974

Reply:
It's coming from this #6039

Reply:
That is a merged PR so the change shouldn't show up here in a diff. I doubt the default value has actually changed in this PR and moreover it definitely should not. What should a pull request about "use needles from correct ref" have to do with changing the default value of git_auto_update?

Reply:
This change was added to commit https://github.com/os-autoinst/openQA/commit/5ee9dd151e20ce0531d27dfd924d21e243055974 and that shouldn't have been the case because it has nothing to do with Git auto update. Note that on current master this setting is also 'no' and none of the commits of this PR should change that (as this PR is not about the Git auto update feature).

Reply:
I'll do a rebase interactive and replace with the version in the master.